ui/packages/consul-ui/app/modifiers/validate.js

```diff
@@ -34,10 +34,10 @@ export default class ValidateModifier extends Modifier {
     }
     if(Object.keys(errors).length > 0) {
       state.context.errors = errors;
-      this.hash.chart.dispatch("ERROR");
+      this.hash.chart.dispatch("ERROR", state.context);
```

Comment:
does it make sense to add a test for this here - or is it better to do this as a follow-up?